### PR TITLE
Skill labels / descriptions

### PR DESCRIPTION
**Pull request by sturatcliffe**

### CODE REVIEW

File: GetJobProfiles/CypherToContentRecipes/CreateSkillContentItems.recipe.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
             "name": "CypherToContent",
             "queries": [
                 {
-                    "query": "MATCH (o:esco__Skill)\r\nreturn { ContentType: 'Skill', TitlePart:{Title:o.skos__prefLabel}\r\n,GraphSyncPart:{Text:o.uri},\r\nSkill:{Description:{Html:'<p>TBC</p>'}}\r\n}\r\norder by o.skos__prefLabel\r\nskip [token:skip] limit [token:limit]"
+                    "query": "MATCH (o:esco__Skill)-[r:dct__description]-(s {esco__language: \"en\"}\r\nreturn { ContentType: 'Skill', TitlePart:{Title:o.skos__prefLabel}\r\n,GraphSyncPart:{Text:o.uri},\r\nSkill:{Description:{Html:s.esco__nodeLiteral}}\r\n}\r\norder by o.skos__prefLabel\r\nskip [token:skip] limit [token:limit]"
```
Comment (lazcool) [inline GetJobProfiles/CypherToContentRecipes/CreateSkillContentItems.recipe.json:16]: Is this missing an `)` for the s node?

Comment (sturatcliffe) [reply GetJobProfiles/CypherToContentRecipes/CreateSkillContentItems.recipe.json:16]: That may well be a copy and paste fail! good spot!

File: GetJobProfiles/CypherToContentRecipes/CreateSkillPrefLabelNodes.recipe.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "website": "",
+    "version": "",
+    "issetuprecipe": false,
+    "categories": "",
+    "tags": [],
+    "steps": [
+        {
+            "name": "CypherCommand",
+            "commands": [
+                "match (n:esco__Occupation) unwind n.skos__prefLabel as prefLabel\r\ncreate (al:SkillLabel:Resource {skos__prefLabel: prefLabel, uri: \"<<ContentApiPrefix>>/skilllabel/\" + apoc.create.uuid()})\r\ncreate (n)-[:hasPrefLabel]->(al)"
```
Comment (lazcool) [inline GetJobProfiles/CypherToContentRecipes/CreateSkillPrefLabelNodes.recipe.json:15]: Should `n` be matching `esco__Skill`?

Comment (sturatcliffe) [reply GetJobProfiles/CypherToContentRecipes/CreateSkillPrefLabelNodes.recipe.json:15]: yep, more copy paste fail!

**Pull request merged**